doc/source/templates/testing/docker/04_finetuning_llms_with_deepspeed/requirements.txt

```diff
@@ -1,4 +1,4 @@
-deepspeed
```

Comment:
does this template work on 0.12.3? Last time we tried it used to have some problems above 0.10.3. Am I remembering a different context?

Reply:
0.12.3 is the one that we install in the ray release byod. So I'd expect it to work.